README.md

```diff
@@ -150,6 +150,7 @@ This setup ensures continuous data synchronization and minimal downtime in captu
 | `publication.tables`                    | []Table  |   yes    |    -    | Set tables which are tracked by data change capture                                                   | Define multiple tables as needed.                                                                                                                  |
 | `publication.tables[i].name`            |  string  |   yes    |    -    | Set the data change captured table name                                                               | Must be a valid table name in the specified database.                                                                                              |
 | `publication.tables[i].replicaIdentity` |  string  |   yes    |    -    | Set the data change captured table replica identity [`FULL`, `DEFAULT`]                               | **FULL:** Captures all columns when a row is updated or deleted. <br> **DEFAULT:** Captures only the primary key when a row is updated or deleted. |
+| `publication.tables[i].schema`          |  string  |    no    |    -    | Set the data change captured table schema name                                                        | Must be a valid table name in the specified database.                                                                                              |
```

Comment:
Can we consider adding a default value for schema as public to maintain backward compatibility? This would ensure existing setups without a specified schema continue to function seamlessly.